fern/docs/pages/guides/ada.mdx

```diff
@@ -39,15 +36,17 @@ from langchain import PromptTemplate
 from sentence_transformers import SentenceTransformer
 import lancedb
 from lancedb.embeddings import with_embeddings
-import predictionguard as pg
+from predictionguard import PredictionGuard
 import pandas as pd
 from getpass import getpass
 ```
 
 ## Authenticate to Prediction Guard API
 ```python
-pg_access_token = getpass('Enter your Prediction Guard access token: ')
-os.environ['PREDICTIONGUARD_TOKEN'] = pg_access_token
+# Set your Prediction Guard token as an environmental variable.
+os.environ["PREDICTIONGUARD_API_KEY"] = "<your access token>"
```

Comment:
Let's use consistent language i.e. use api key everywhere
```suggestion
os.environ["PREDICTIONGUARD_API_KEY"] = "<api key>"
```

Reply:
Thanks for feedback @michalnicp . Have changed all such instances to <api key. to maintain consistency.